documentation/src/docs/deployment/run-local.md

```diff
@@ -76,9 +79,13 @@ HOST_IP=1.2.3.4 docker-compose up
 ```
 Clients can then connect to the controller at `${HOST_IP}:9090`.
 
-## Configuring standalone
+## Running Pravega in Standalone Mode with SSL/TLS Enabled
 
-Configure standalone server to communicate using SSL/TLS. To do so, edit the TLS-related properties in `standalone-config.properties` as shown below:
+By default the `singlenode.enableTls` (SSl/TLS) is disabled. To enable it, set the value of  `singlenode.enableTls` to **True**. Once the `enableTls` (TLS) is enabled, the default certificates provided in the `conf` directory are used for setting up TLS.
+
+Set the following configuration properties:
+
+1. Configure standalone server to communicate using SSL/TLS. To do so, edit the TLS-related properties in `standalone-config.properties` as shown below:
 
   ```java
   singlenode.enableTls=true
```

Comment:
Please correct the indentation for the text and code under these numbered items of the list. For example, the formatted configuration code should start at the same location where the text for the numbered item starts.

Reply:
@ravisharda Sure will edit

Reply:
Accepted suggestion

Reply:
Indentation is not corrected in the latest commit.